libhb/handbrake/preset_builtin.h

```diff
@@ -10042,7 +10042,7 @@ const char hb_builtin_presets_json[] =
 "            \"Folder\": false,\n"
 "            \"FolderOpen\": false,\n"
 "            \"InlineParameterSets\": false,\n"
-"            \"MetadataPassthrough\": true,\n"
+"            \"MetadataPassthru\": true,\n"
```

Comment:
We'll need to migrate old->new for this option

Reply:
Got it!